beacon-chain/core/epoch/participation.go

```diff
@@ -0,0 +1,35 @@
+package epoch
```

Comment:
Makes more sense to have this key function in core for reuse, given it is a pure function